src/components/BulkEditPane/BulkEditListResult/BulkEditMark/BulkEditMarkForm/BulkEditMarkForm.js

```diff
@@ -29,6 +29,18 @@ const BulkEditMarkForm = () => {
 
   const isIndicatorDirty = (value) => value?.length && value !== '\\';
 
+  const isValueFieldValid = (value) => {
+    if (Number(value.trim()).toString().length === 3) {
+      return isMarcValueValid(value);
+    } else return true;
```

Comment:
else is redundant, can be just 

`return true;`

Reply:
done